### DISCUSSION TITLE
More API Endpoints

### DISCUSSION THREAD

**Discussion by kimfom01** [postalserver/postal#3171] [Development discussions]
Are there any plans to add API for programmatic config and setup of Postal?  
An example of this would be using the API to create servers and adding/verifying domains etc.

**Comment by willpower232** [comment]
I believe there is a new API in the works somewhere but haven't heard about it for a while I'm afraid.

**Reply by kimfom01** [reply]
On a scale of 1 - 10 how difficult will it be for a non ruby dev to extend postal and expose the endpoints?

I'm considering going ahead to do it myself and build / manage the docker images until an official release is out

**Comment by z3d3m0n** [comment]
great idea , i am willing to contribute manpower to help with this tak
